src/main/java/edu/cornell/kfs/sys/util/CuAttachmentUtils.java

```diff
@@ -0,0 +1,43 @@
+package edu.cornell.kfs.sys.util;
+
+import java.net.URLConnection;
+import java.util.Locale;
+
+import org.apache.commons.lang3.StringUtils;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.kuali.kfs.sys.KFSConstants;
+
+public final class CuAttachmentUtils {
```

Comment:
There is a find mime type function in AccountingXmlDocumentDownloadAttachmentServiceImpl

Reply:
As noted in the file's comments, I specifically copied that method's logic into this file. Should I create a future user story to merge our MIME type handling into a single unified location?